eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -5,24 +5,39 @@
   },
   "matrix": {
     "Agent": {
-      "ubuntu-20.04": {
-          "OSVmImage": "MMSUbuntu20.04",
-          "Pool": "azsdk-pool-mms-ubuntu-2004-general"
+      "Ubuntu-20.04_NETCore3.1": {
+        "OSVmImage": "MMSUbuntu20.04",
+        "Pool": "azsdk-pool-mms-ubuntu-2004-general",
+        "TestTargetFramework": "netcoreapp3.1"
       },
-      "windows-2019": {
-          "OSVmImage": "MMS2019",
-          "Pool": "azsdk-pool-mms-win-2019-general"
+      "Ubuntu-20.04_NET6.0": {
+        "OSVmImage": "MMSUbuntu20.04",
+        "Pool": "azsdk-pool-mms-ubuntu-2004-general",
+        "TestTargetFramework": "net6.0"
       },
-      "macOS-10.15": {
-          "OSVmImage": "macOS-10.15",
-          "Pool": "Azure Pipelines"
+
+      "Windows2019_NET6.0": {
+        "OSVmImage": "MMS2019",
+        "Pool": "azsdk-pool-mms-win-2019-general",
+        "TestTargetFramework": "net6.0"
```

Comment:
Is netcoreapp3.1 not included here because it is the default below?

Reply:
Yep, we cover all 3 frameworks on windows this way.

Reply:
Where is the entry with the default?

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/26512/files#diff-6d57a4119ae22b0fede4d3aab5dff565cee910416d4eaa647d83e1d749c9b559R50